bb_wrapper/wrapper/pagamento_lote.py

```diff
@@ -528,6 +530,7 @@ def criar_transferencia_pix(
         chave,
         descricao="",
         tipo_pagamento=128,
+        documento="",
```

Comment:
Pode usar o None mesmo. Acredito que seja um valor mais fácil de identificar que é vazio semanticamente para quem ler o código da lib.

```suggestion
        documento=None,
```